lib/cretonne/src/ir/entities.rs

```diff
@@ -23,7 +23,7 @@ use std::fmt;
 use std::u32;
 
 /// An opaque reference to an extended basic block in a function.
-#[derive(Copy, Clone, PartialEq, Eq, Hash, Debug, PartialOrd, Ord)]
+#[derive(Copy, Clone, PartialEq, Eq, Hash, Debug, PartialOrd, Ord, Default)]
```

Comment:
This is probably similar to the problem with the `Layout::entry_block()` method. The solution is to use an `Option<Ebb>`.

Implementing `Default` for `Ebb` indicates to users that `ebb0` is somehow a sensible default, but it really isn't.